app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -0,0 +1,226 @@
+
+If you’re not sure of the status of your visa, check:
+
+* [Ukrainian nationals in the UK: visa support](https://www.gov.uk/guidance/ukrainian-nationals-in-the-uk-visa-support)
+* [UK visa support for Ukrainian nationals](https://www.gov.uk/guidance/support-for-family-members-of-british-nationals-in-ukraine-and-ukrainian-nationals-in-ukraine-and-the-uk)
+* [Apply to stay in the UK under the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
+
+If you are already in the UK under a [different type of visa](https://www.gov.uk/guidance/ukrainian-nationals-in-the-uk-visa-support) (for example, a visitor or seasonal worker visa) you can apply to the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme). Depending on your personal circumstances, you could also apply for a [skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration) (to work as a teacher) or [student visa](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa) (to train as a teacher). 
+
+You will need to meet the eligibility requirements for these types of visas and you should be aware that time spent in the UK under the Ukraine Extension Scheme and the student visa does not count towards an application for permanent settlement. Time spent in the UK under the skilled worker visa does count towards such an application. 
+
+### Useful documentation for teachers and trainee teachers
+
+
+For safeguarding reasons, schools will assess your suitability to teach from a range of different sources, including:
+
+
+* 2 references from previous employers
+* your complete employment history
+* evidence of your professional standing in Ukraine – for example, documentation from the [Ministry of Education and Science](https://mon.gov.ua/ua) showing you are qualified and have no sanctions against you as a teacher
+* a criminal record check
+
+Teacher training providers will ask for:
+
+* evidence that applicants for teacher training have a bachelor’s degree from a university (non-UK or in the UK)
+* a criminal record check
+
+You can apply to the [Ministry of Internal Affairs of Ukraine](https://vytiah.mvs.gov.ua/app/landing) for a criminal record check, which will be emailed to you. You can then contact the [Ukrainian Embassy in the UK](https://vytiah.mvs.gov.ua/app/landing) for a translation of the criminal record check and a letter confirming its validity.
```

Comment:
@laurahermionetennant92 The two links in this sentence go to the same page which is flagged by our accessibility checker as unnecessary

Once the page has gone live, I'd suggest either changing the links to go to different locations or removing the second link?

Reply:
Thank you that's a mistake, the second link should to go the Ukrainian embassy in London, will fix